core/src/main/kotlin/behavior/ChatInputCommandBehavior.kt

```diff
@@ -16,8 +18,9 @@ public interface ChatInputCommandBehavior : ApplicationCommandBehavior {
 
 public interface GuildChatInputCommandBehavior : ChatInputCommandBehavior, GuildApplicationCommandBehavior {
 
+     @OptIn(KordUnsafe::class)
      override suspend fun edit(builder: suspend ChatInputModifyBuilder.() -> Unit): GuildChatInputCommand {
-          val request = ChatInputModifyBuilder().apply { builder() }.toRequest()
+          val request = ChatInputModifyBuilderImpl().apply { builder() }.toRequest()
           val response = service.modifyGuildApplicationCommand(applicationId, guildId, id, request)
```

Comment:
```suggestion
          val response = service.modifyGuildChatInputApplicationCommand(applicationId, guildId, id) { builder() }
```
allows `ChatInputModifyBuilderImpl` to be internal